pkg/apis/servicecatalog/types.go

```diff
@@ -365,10 +372,20 @@ type BindingSpec struct {
 	// Immutable.
 	InstanceRef v1.LocalObjectReference
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is a set of the parameters to be
 	// passed to the underlying broker.
+	// The inline YAML/JSON payload to be translated into equivalent
+	// JSON object.
```

Comment:
Yeah, we need to document that you will get an error if a key is declared twice among the inline parameters and the parameters taken from secret keys in blobs of json.  This should be mentioned in the godoc for both the `Parameters` and `ParametersFrom` fields.